hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNodeRpcServer.java

```diff
@@ -979,7 +979,10 @@ public boolean complete(String src, String clientName,
                           ExtendedBlock last,  long fileId)
       throws IOException {
     checkNNStartup();
-    return namesystem.completeFile(src, clientName, last, fileId);
+    boolean result = namesystem.completeFile(src, clientName, last, fileId);
+    LOG.debug("*DIR* NameNode.complete: src={}, clientName={}, fileId={}, result={}.",
+        src, clientName, fileId, result);
+    return result;
```

Comment:
The `namesystem.completeFile` already logs in into the stateChange log
```
      NameNode.stateChangeLog.info("DIR* completeFile: " + src
          + " is closed by " + holder);
```

I don't think you need this change in this scope. In general complete changes the Fs state so it uses ``stateChangeLog`` if any API changes block state it uses ``blockStateChangeLog`` for rest snapshot or EC operations it is normal log

Reply:
@ayushtkn Master, thanks for your review.  I have updated this PR, please help me review it again when you are available. Thanks